### ISSUE TITLE
Implement (support for) gene family linkout service

### ISSUE THREAD

**Issue by alancleary** [legumeinfo/lis_context_viewer#290]
The gene family detail component is using esoteric knowledge of the Legume Information System to create links to gene family trees (though the URL is provided via the config file, the GET structure of the URL is hard coded into the component's template). This is an issue for two reasons: 1) It tightly couples the GCV with the LIS gene family trees and 2) if people not affiliated with LIS want to use link out to the families they're using, they either have to match the URL structure used by LIS or change this hard coding.

For these reasons, the component should be updated to get tree URL (and any other family related URLs) via a service, as is being done with the gene link out service.

Perhaps this should be wrapped into an epic with issue #129, which aims to do the same thing but for the micro-track detail component.

**Comment by adf-ncgr** [comment]
I agree with the sentiment expressed here, though do not thereby wish to imply that esoteric knowledge of anything leguminous is dishonorable.
